docs/Contributing/Releasing-Fleet.md

```diff
@@ -1,158 +1,168 @@
 
-5. Create a [Release QA](https://github.com/fleetdm/fleet/blob/main/.github/ISSUE_TEMPLATE/smoke-tests.md) issue. Populate the version and browsers, and assign to the QA person leading the release. Add the appropriate [product group label](https://fleetdm.com/handbook/company/product-groups), and `:release` label, so that it appears on the product group's release board.
+7. Tag and push the new release in Git:
+  
+```sh
+git tag fleet-v-v4.3.1
+git push origin fleet-v-4.3.1
+```
+
+Note that `origin` may be `upstream` depending on your `git remote` configuration. The intent here
+is to push the new tag to the `github.com/fleetdm/fleet` repository.
+
+After the tag is pushed, GitHub Actions will automatically begin building the new release.
+
+***
+
+Wait while GitHub Actions creates and uploads the artifacts.
+
+***
 
-6. Ask QA to conduct release tests. When they complete, the release is ready to publish.
+When the Actions Workflow has been completed, [publish the new version of Fleet](#publish-a-new-version-of-fleet).
 
-7. **Important!** The DRI for creating the patch release branch manually checks the database migrations. Any migrations that are not cherry-picked in a patch must have a _later_ timestamp than migrations that were cherry-picked. If there are new migrations that were not cherry-picked, verify that those migrations have later timestamps. If they do not, submit a new Pull Request to increase the timestamps and ensure that migrations are run in the appropriate order.
+### Create release candidate
 
-8. Tag and push the new release in Git:
+1. Push a branch containing new commits to [fleetdm/fleet](https://github.com/fleetdm/fleet) that begins with `prepare-*` or `patch-*`. 
    ```sh
-   git tag fleet-v-v4.3.1
-   git push origin fleet-v-4.3.1
+   git push origin patch-fleet-v4.3.1
    ```
 
-   Note that `origin` may be `upstream` depending on your `git remote` configuration. The intent here
-   is to push the new tag to the `github.com/fleetdm/fleet` repository.
+   > When a `prepare-*` or `patch-*` branch is pushed, the [Docker publish Action](https://github.com/fleetdm/fleet/actions/workflows/goreleaser-snapshot-fleet.yaml) will run and create a container image for QA with `fleetctl preview` (eg. `fleetctl preview --tag patch-fleet-v4.3.1`).
+
+5. Check the [Docker Publish GitHub action](https://github.com/fleetdm/fleet/actions/workflows/goreleaser-snapshot-fleet.yaml) to confirm it completes successfully for this branch.
 
-   After the tag is pushed, GitHub Actions will automatically begin building the new release.
+5. Create a [Release QA](https://github.com/fleetdm/fleet/blob/main/.github/ISSUE_TEMPLATE/smoke-tests.md) issue. Populate the version and browsers, and assign to the QA person leading the release. Add the appropriate [product group label](https://fleetdm.com/handbook/company/product-groups), and `:release` label, so that it appears on the product group's release board.
```

Comment:
```suggestion
3. Create a [Release QA](https://github.com/fleetdm/fleet/blob/main/.github/ISSUE_TEMPLATE/smoke-tests.md) issue. Populate the version and browsers, and assign to the QA person leading the release. Add the appropriate [product group label](https://fleetdm.com/handbook/company/product-groups), and `:release` label, so that it appears on the product group's release board.
```